src/System.Text.Json/tests/Serialization/Value.ReadTests.cs

```diff
@@ -218,18 +215,20 @@ public static void RangePass()
         }
 
         [Fact]
-        [ActiveIssue(37992)]
-        // These conditions should be merged in with RangePass once issues identified.
-        public static void RangePassFloatingFail()
+        [ActiveIssue(37992, TargetFrameworkMonikers.NetFramework)]
+        public static void RangePassFloatingPoint()
         {
 #if netcoreapp // SingleToInt32Bits not present on netfx (we could also shim that method)
             Assert.Equal(0x00000000u, (uint)BitConverter.SingleToInt32Bits(JsonSerializer.Parse<float>("0")));
             Assert.Equal(0x80000000u, (uint)BitConverter.SingleToInt32Bits(JsonSerializer.Parse<float>("-0")));
             Assert.Equal(0x80000000u, (uint)BitConverter.SingleToInt32Bits(JsonSerializer.Parse<float>("-0.0")));
 #endif
 
-            Assert.Equal(float.MaxValue, JsonSerializer.Parse<float>(float.MaxValue.ToString()));
-            Assert.Equal(float.MaxValue, JsonSerializer.Parse<float?>(float.MaxValue.ToString()));
+            Assert.Equal(float.MaxValue, JsonSerializer.Parse<float>(float.MaxValue.ToString(CultureInfo.InvariantCulture)));
```

Comment:
Full framework has multiple issues formatting and parsing floating-point values and can't actually ever guarantee roundtripping.

Reply:
So that is probably part of the issues you are seeing.

It also has a different behavior when dealing with values it thinks are too large and will throw an exception rather than return an infinity with the appropriate sign.